src/style.css

```diff
@@ -19,7 +19,7 @@ input,
 button {
   font-size: 25px;
   border: 7px solid black;
-  padding: 12px;
+  padding: 11px;
 }
 
```

Comment:
- [ ] Good job so far but your app is not working as expected :cry: Your application displays some Javascript code  **[object HTMLInputElement]:[object Object]** as you can see in the screenshots. Please fix that to make your app work as expected. :+1: 

![Screenshot from 2022-08-25 13-06-49](https://user-images.githubusercontent.com/6615394/186675196-d26fc65f-5641-47e3-91ee-8a6afc3af300.jpg)

- [ ] I would like to recommend you arrange your app design. Please try to stick to the wireframe as much as possible cause it is very important for the developer to peer attention to details :smile: 
You can check the screenshots below to see the differences :arrow_down: 

#### Wireframe
![Screenshot from 2022-08-25 13-17-29](https://user-images.githubusercontent.com/6615394/186676456-b5e33c14-adfc-4969-b5c4-2c28e1269052.png)

#### Your design
![Screenshot from 2022-08-25 13-17-56](https://user-images.githubusercontent.com/6615394/186676580-236dcdfc-4f9a-4d6f-9757-c5aa6c178337.jpg)

Reply:
@codecaiine For [object HTMLInputElement]:[object Object] I can't delete that as it already post in API and delete action is not available in that given URL